docs/glossary.md

```diff
@@ -0,0 +1,86 @@
+# Glossary of Terms and Concepts
+
+## 1. **Event**
+- **Definition:** A single observation or occurrence within a temporospatial grid cell. In this context, an event refers to the recorded value of fatalities per 100k population in a specific grid cell during a specific time period.
+- **Example:** An event could be the number of fatalities per 100k population recorded in a grid cell during a specific month.
+
+## 2. **(PRIO) Grid Cell**
+- **Definition:** The smallest spatial unit in the analysis, defined by specific geographic coordinates and time intervals. It is the basic unit used to measure events (e.g., fatalities per 100k population).
+- **Example:** A PRIO grid cell is a $0.5 \times 0.5$ decimal degrees geographic area observed over a month.
+
+## 3. **Time Period**
+- **Definition:** The temporal component of the grid cell. This could be a month, year, or any other defined period.
+- **Example:** A time period could be a month when measuring monthly fatalities per 100k population.
+
+## 4. **Potential Event**
+- **Definition:** Any grid cell within the defined geographic and temporal scope that could potentially contain an event.
+- **Example:** If a country is divided into 100 grid cells and observed over 400 months, there are 40,000 potential events.
```

Comment:
Good example for understanding! Would not have thought about doing this. Consider: explicit language stating how this is an example at the country-level that is dependent on temporal and geographic scale.